src/style.css

```diff
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: inline-flex;
+  flex-direction: column;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
```

Comment:
You don't need to add this rule for every modifier. You can just place it inside `.stars` selector if you need it everywhere.